### ISSUE TITLE
Set class for embedded content in basic WYSIWYG editor

### ISSUE THREAD

**Issue by choffa** [flamelink/flamelink#99]
While working on a flamelink solution for a client, we wanted to use the basic WYSIWYG editor for article content. Now we want to allow some basic embedding of content (such as Youtube `iframe`). We want to be able to apply some basic styling to this content on our frontend (A Gatsby.js page if that matters), however, there is no way to set a class for such content in flamelink. When editing the source everything except `height`, `width` and `src` is purged, while `frameBorder` is forced to `frameBorder="0"`.

# What we tried
We considered using either the CKEditor or Markdown editors. However, we very much want the drag and drop image functionality of the basic editor.

# Request
Add the possibility to set class for embedded content in the WYSIWYG editor, either when embedding or by editing the source.

**Comment by gitdubz** [comment]
Thanks @choffa for the detailed explanation. We will investigate to see if this is something that can be configured within the WYSIWYG and get back to you.

**Comment by gitdubz** [comment]
Hi @choffa we have tried to implement a fix for the default WYSIWYG editor but the editor was not designed to be used in this way. We found quite a couple of open requests to add this functionality to the editor, and exhausted all the possible solutions suggested on forums but to no avail.

We recommend switching to the CKEditor version.
A button has been added above the editor to easily access the media manager in order to copy a file url to insert into the editor.

![Image 2020-10-13 at 9 23 59 AM](https://user-images.githubusercontent.com/48430891/95829002-f896eb00-0d35-11eb-917b-99a1059046b6.png)
